README.md

```diff
@@ -86,3 +86,20 @@ The test results will output to the terminal.
 
 For Prod - http://localhost
 For Dev - http://localhost:5000
+
+To run the commands manually you can use:
+docker build --target test --tag todo-app:test . 
+docker run todo-app:test tests
+docker run -e TRELLO_API_KEY=${{ secrets.TRELLO_API_KEY }} -e TRELLO_API_SECRET=${{ secrets.TRELLO_API_SECRET }} -e TRELLO_BOARD_ID=${{ secrets.TRELLO_BOARD_ID }}  todo-app:test tests_e2e
```

Comment:
Don't forget that when running this manually this won't quite work because the secrets won't make any sense locally! Instead we _could_ advise passing each env var through directly, but really we'll want to use that env-file argument:
```suggestion
docker run --env-file .env  todo-app:test tests_e2e
```

Reply:
Ah yes, I've updated my read me on the current module so going forward this will be correct.  I'll just merge this now though.